### DISCUSSION TITLE
The following resource(s) failed to create: [RDSInstance].

### DISCUSSION THREAD

**Discussion by rshettyami** [awslabs/aws-saas-boost#489] [Q&A]
I am seeing following error when I attempt to provision a tenant in AWS SaaS Boot.

#####################
The following resource(s) failed to create: [RDSInstance].
######################

Has anyone encountered this issue?

**Comment by PoeppingT** [comment]
Hello @rshettyami , there are many reasons why you might see this error in CloudFormation. The best way to investigate further is to inspect the failed stack directly in the CloudFormation UI, click on the "Events" tab at the top, and search for the `CREATE_FAILED` event for the `RDSInstance` resource. There CloudFormation will show you the actual failure.
